lib/SILOptimizer/Utils/Devirtualize.cpp

```diff
@@ -192,19 +182,6 @@ static bool isKnownFinalClass(ClassDecl *CD, SILModule &M,
   // of devirtualization.
   if (CHA) {
     if (!CHA->hasKnownDirectSubclasses(CD)) {
-      switch (CD->getEffectiveAccess()) {
-      case AccessLevel::Open:
-        return false;
-      case AccessLevel::Public:
-      case AccessLevel::Internal:
-        if (!M.isWholeModule())
-          return false;
-        break;
-      case AccessLevel::FilePrivate:
-      case AccessLevel::Private:
-        break;
-      }
-
```

Comment:
The above code also appeared on lines 174 to 185, so it seems redundant to duplicate them here.